__tests__/src/reducers/windows.test.js

```diff
@@ -121,6 +121,42 @@ describe('windows reducer', () => {
     });
   });
 
+  describe('SET_WINDOW_COMPANION_WINDOW', () => {
+    it('sets the given type under the given position when no companion window exists', () => {
+      const action = {
+        type: ActionTypes.SET_WINDOW_COMPANION_WINDOW,
+        windowId: 'abc123',
+        position: 'right',
+        panelType: 'info',
+      };
+      const before = {
+        abc123: {},
+      };
+      const after = {
+        abc123: { companionWindows: { right: 'info' } },
```

Comment:
Note the shape of the state here.  I'm not sold on this working long-term, but it's good enough for the acceptance criteria as outlined and would allow us to move the thumbnail content there as well.  (It would not allow for multiple companion windows to be open in the same right/bottom position though).